interfaces/cython/cantera/reaction.pyx

```diff
@@ -194,7 +194,7 @@ cdef class ArrheniusRate(ArrheniusTypeRate):
 
     .. math::
 
-        k_f = A T^b \exp{-\tfrac{E}{RT}}
+        k_f = A T^b \exp(-\tfrac{E}{RT})
 
     where ``A`` is the `pre_exponential_factor`, ``b`` is the `temperature_exponent`,
     and ``Ea`` is the `activation_energy`.
```

Comment:
Changing the equation to "E_a" instead of just "E" would make sense, but I think formatting these as code makes sense for the correspondence with the constructor arguments.

Reply:
That's fine, but then we should be consistent in all the reaction-type docstrings. The `TwoTempPlasmaRate` uses math to show these parameters.

Reply:
Can I suggest that the docstring for `TwoTempPlasmaRate` is out-of-scope for this PR? There are other updates needed there that I would like @BangShiuh to address anyway.

Reply:
Sounds good to me!